renderer.go

```diff
@@ -128,7 +130,7 @@ type htmlRenderer struct {
 }
 
 func (mr *htmlRenderer) Apply(template *Template) ([]byte, error) {
-	tmpl, err := html_template.New("Text Template").Funcs(funcMap).Parse(mr.inputTemplate)
+	tmpl, err := html_template.New("Text Template").Funcs(funcMap).Funcs(sprig.TxtFuncMap()).Parse(mr.inputTemplate)
```

Comment:
/nit I know they both return `GenericFuncMap` [(source)](https://github.com/Masterminds/sprig/blob/6b2a58267f6a8b1dc8e2eb5519b984008fa85e8c/functions.go#L46-L53), but just to be clearer and protect against future changes, can we make this one use `sprig.HtmlFuncMap()`